Modix.Services/Moderation/ModerationService.cs

```diff
@@ -528,7 +528,7 @@ private async Task<IRole> GetOrCreateDesignatedMuteRoleAsync(IGuild guild, ulong
                     return guild.Roles.First(x => x.Id == mapping.Role.Id);
 
                 var role = guild.Roles.FirstOrDefault(x => x.Name == MuteRoleName)
-                    ?? await guild.CreateRoleAsync(MuteRoleName);
+                           ?? await guild.CreateRoleAsync(MuteRoleName);
```

Comment:
Why the space here? A continuation of a statement should be indented 1 level deeper than the line on which the statement starts.

Reply:
This can be removed, must have hit tab by mistake